### PR TITLE
[FrameworkBundle] Use class constant when possible

### PR DESCRIPTION
**Pull request by dunglas**
| Q | A |
| --- | --- |
| Branch? | 3.1 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php [hunk 1]
```diff
@@ -28,6 +32,7 @@
 use Symfony\Component\Serializer\Normalizer\DataUriNormalizer;
 use Symfony\Component\Serializer\Normalizer\DateTimeNormalizer;
 use Symfony\Component\Serializer\Normalizer\JsonSerializableNormalizer;
+use Symfony\Component\Validator\Mapping\Cache\CacheInterface;
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: wrong use statement. you selected the wrong cache interface

Comment (dunglas) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: thanks, fixed.

**Comment by fabpot** [comment]
I don't think this is worth it. It does not change anything and we have many other places that we could potentially change.

**Comment by dunglas** [comment]
IIRC we acted to always use class constants when possible for Symfony >=3.0. The rationale behind this PR was to follow our best practices :)

I agree that it's a minor change, but it still helps when using an IDE and IDE (find usages, refactoring tools...).

**Comment by fabpot** [comment]
But as of today, we are not doing this everywhere. So, we can either change all occurrences or just use the new best practice for new code.

**Comment by dunglas** [comment]
Ok, closing for now.

**Pull request closed without merging**